packages/stryker/src/reporters/ScoreReporter.ts

```diff
@@ -0,0 +1,70 @@
+import {Reporter, ScoreResult} from 'stryker-api/report';
+import StrykerBadgeClient from './ScoreReporter/StrykerBadgeClient';
+import {getEnvironmentVariable} from '../utils/objectUtils';
+import { getLogger } from 'log4js';
+import { StrykerOptions } from 'stryker-api/core';
+
+export default class ScoreReporter implements Reporter {
+  private readonly log = getLogger(ScoreReporter.name);
+
+  constructor(
+    setting: StrykerOptions,
+    private strykerBadgeClient: StrykerBadgeClient = new StrykerBadgeClient()
+  ) { }
+
+  private isPullRequest: boolean;
+  private repository: string | undefined;
+  private branch: string | undefined;
+  private apiKey: string | undefined;
+  private mutationScore: number;
+
+  private readEnvironmentVariables() {
+    const isPullRequest = getEnvironmentVariable('TRAVIS_PULL_REQUEST') !== 'false';
+    if (isPullRequest) {
+      this.log.warn('Pull requests will not be reported to the badge report server');
```

Comment:
We should only read env variables if `TRAVIS=true` (https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables). That indicates that the build is running travis and thus it is a warning of these variables are missing. 

If you would run the build locally, it would be enough to log as debug that the `TRAVIS` env variable is missing and reporting is skipped.

Reply:
Done